### PR TITLE
ankit2022counterfactual 

### PR DESCRIPTION
**Pull request by Ankit2527**
## Project Name

Write the `ankit2022counterfactual` here, and as well as rename the PR name to reflect the `<paper_citekey>` for quick reviewing. 

## Compiled PDF
Add the link to the compiled pdf here. Example : [Compiled Pdf](2021/ankit2022counterfactual/journal/article.pdf)

## Metadata Checklist

Ensure that you have added the following metadata information (put an x to the checkbox items) in `<paper_citekey>/journal/metadata.yaml`.

- [x] `title` field exists and it starts with `[Re] `.
- [x] `authors` contain author information with affiliations.
- [x] [ORCID id](https://orcid.org/) is provided for authors.
- [x] One contact author is added using `*`.
- [x] Github `url` is provided in `code` section.
- [x] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x] `replication` section is populated with the necessary information about the paper you reproduced.
- [x] `abstract` field is populated with your paper abstract (including double quotes)
- [x] `domain` should contain "ML Reproducibility Challenge 2021"
- [x] `review` section contains the url to the OpenReview forum of your paper.
- [x] Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors

If your build had any errors locally, please add it here. You can upload your build.log in a [Pastebin](https://pastebin.com/) link and put it here.

### CODE REVIEW

**Comment by koustuvsinha** [comment]
Hi, the build is failing as you have mentioned multiple contact authors. Please update your PR!

> AssertionError: Only one contact author needs to be specified

**Comment by koustuvsinha** [comment]
Hi, please ensure all items in the checklist is complete. If they are, please mark it accordingly in the above comment.

**Comment by Ankit2527** [comment]
Hi @koustuvsinha I am not able to get the appendix in the generated pdf, I have created a separate appendix.tex file, also added input<appendix.tex> in main.tex. Can you help me out with it?

**Comment by Ankit2527** [comment]
Hi @koustuvsinha, the issue with appendix is fixed. Need to make some minor change, will try to push after those changes asap.

File: 2021/ankit2022counterfactual/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+% to compile a camera-ready version, add the [final] option, e.g.:
+%\usepackage[]{neurips_2019}
+
+% to avoid loading the natbib package, add option nonatbib:
+%     \usepackage[nonatbib]{neurips_2019}
+
+% \usepackage[hyphens,spaces,obeyspaces]{url} (This causes issues, don't use this)
+%\usepackage[colorlinks,allcolors=blue]{hyperref}
+
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```
Comment (koustuvsinha) [inline 2021/ankit2022counterfactual/openreview/packages.tex:14]: Please comment this line (reasoning in README)

**Pull request merged**